CenterForOpenScience/osf.io
# [#PLAT-366]Add download counts to admin

## Purpose
Investigate why the keen download counts are way lower than the data in the pagecounter. And find out that the keen download counts reflected in admin app only gives the download count for yesterday. Add total download counts to admin.


## Changes

- [x] check the reason that keen's file download data is way different from the data in pagecounter

- [x] write script to calculate the file download counts and store it somewhere in the database

- [x] change the metrics.es6.js to reflect the total download counts



## QA Notes
Now when you log on to admin app in the OSF Metrics page, check for the downloads number, you should be able to see the total file download counts and total file unique download counts. And the chart data from keen's title has been adjusted to 'The total number of files downloaded from the OSF for yesterday' and 'The number of unique downloads of files on the OSF for yesterday'.


## Side Effects
Don't have keen api locally, cannot really test this locally. And also this is trying to initial a keen data storage backend model as from ticket https://openscience.atlassian.net/browse/PRODUCT-28. Currently all the metrics data from keen on admin app is fetched from keen each time that page is rendered which is not ideal, since we are paying keen for the query number. Later on it will requires an improvement to have a full set of data storage backend model so we can just pull all those numbers from database.


## Ticket
https://openscience.atlassian.net/browse/PLAT-366
